src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,32 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Transaction transaction = null;
+        try (Session session = factory.openSession()) {
```

Comment:
don't use try with resources and transaction, rollback won't happen because session will be closed before catch block